### PR TITLE
Details: Show pre-evolutions

### PR DESCRIPTION
**Pull request by Distrib-ps**
PR for : https://www.smogon.com/forums/threads/show-previous-evolution-in-dt-pok%C3%A9mon.3701062/ 

image

### CODE REVIEW

File: server/chat-commands/info.ts [hunk 1]
```diff
@@ -630,6 +630,7 @@ export const commands: Chat.ChatCommands = {
 					};
 					details["Weight"] = `${pokemon.weighthg / 10} kg <em>(${weighthit} BP)</em>`;
 					const gmaxMove = pokemon.canGigantamax || dex.species.get(pokemon.changesFrom).canGigantamax;
+					const preEvolution = dex.species.get(newTarget.name).prevo;
```
Comment (Karthik99999) [inline server/chat-commands/info.ts:633]: Is there a reason you didn't just do `pokemon.prevo`?

Comment (Distrib-ps) [reply server/chat-commands/info.ts]: No, there was no particular reason. `pokemon.prevo` is correct, thx

File: server/chat-commands/info.ts [hunk 2]
```diff
@@ -630,6 +630,7 @@ export const commands: Chat.ChatCommands = {
 					};
 					details["Weight"] = `${pokemon.weighthg / 10} kg <em>(${weighthit} BP)</em>`;
 					const gmaxMove = pokemon.canGigantamax || dex.species.get(pokemon.changesFrom).canGigantamax;
+					const preEvolution = pokemon.prevo;
```
Comment (Karthik99999) [inline server/chat-commands/info.ts:633]:
```suggestion
```

File: server/chat-commands/info.ts [hunk 3]
```diff
@@ -665,6 +666,9 @@ export const commands: Chat.ChatCommands = {
 							}
 						}
 					}
+					if (preEvolution) {
+						details["Pre-Evolution"] = preEvolution;
+					}
```
Comment (Karthik99999) [inline server/chat-commands/info.ts:671]:
```suggestion
					if (pokemon.prevo) {
						details["Pre-Evolution"] = pokemon.prevo;
					}
```

**Comment by KrisXV** [comment]
looks good, thanks!

**Pull request merged**